conda_build_config.yaml

```diff
@@ -17,35 +17,35 @@ python:
 python_impl:
   - cpython
 
-# Project overrides
-macos_min_version:             # [osx and x86_64]
-  - 10.14                      # [osx and x86_64]
-macos_machine:                 # [osx]
-  - x86_64-apple-darwin13.4.0  # [osx and x86_64]
-  - arm64-apple-darwin20.0.0   # [osx and arm64]
-MACOSX_DEPLOYMENT_TARGET:      # [osx]
-  - 11.0                       # [osx and arm64]
-  - 10.14                      # [osx and x86_64]
-CONDA_BUILD_SYSROOT:
-  - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk  # [osx and arm64]
+# # Project overrides
+# macos_min_version:             # [osx and x86_64]
+#   - 10.14                      # [osx and x86_64]
+# macos_machine:                 # [osx]
+#   - x86_64-apple-darwin13.4.0  # [osx and x86_64]
+#   - arm64-apple-darwin20.0.0   # [osx and arm64]
+# MACOSX_DEPLOYMENT_TARGET:      # [osx]
+#   - 11.0                       # [osx and arm64]
+#   - 10.14                      # [osx and x86_64]
+# CONDA_BUILD_SYSROOT:
+#   - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk  # [osx and arm64]
 
 
 # fix build metadata, needed for mapviz and moveit-core
-replacements:
```

Comment:
This section has no equivalent yet in the new format. However, I am planning to add this functionality in the `recipe.yaml` rather than in the variant config because I don't understand why this was ever placed in that file in the first place.